bullet_train-themes-light/app/views/showcase/previews/field_partials/attribute_partials/_about_attribute_partials.html.erb

```diff
@@ -0,0 +1,19 @@
+<% showcase.description do %>
+  Attribute partials are used to display a single attribute of a model, within an `index` table colum or the resource's `show` screen.
+
+  Read more about them and about the form field partials in the extensive <a href="https://bullettrain.co/docs/field-partials" target="_blank">developer documentation</a>.
+<% end %>
+
+<% if partial.body? %>
+  <%= partial.body.yield Scaffolding::CompletelyConcrete::TangibleThing.new %>
+<% end %>
```

Comment:
We've also got an `optional` proxy that only calls the method if `body?`, like this:

```suggestion
<%= partial.body.optional.yield Scaffolding::CompletelyConcrete::TangibleThing.new %>
```